compiler/x/codegen/OMRCodeGenerator.cpp

```diff
@@ -365,12 +365,12 @@ OMR::X86::CodeGenerator::initialize(TR::Compilation *comp)
    self()->setSupportsArrayCmp();
    self()->setSupportsArrayCopy();
 
-   if (comp->getOption(TR_EnableX86AdvancedMemorySet))
-      self()->setSupportsArraySet();
-
    static char *disableArraySet = feGetEnv("TR_disableArraySet");
```

Comment:
Okay, using an option can be a valuable debugging aid where you can narrow down to a particular method without needing to re-build. @andrewcraik what is your opinion ?